src/area-chart/chart-container.tsx

```diff
@@ -78,6 +78,7 @@ function ChartContainer<T extends AreaChartProps.DataTypes>({
 }: ChartContainerProps<T>) {
   const [leftLabelsWidth, setLeftLabelsWidth] = useState(0);
   const [containerWidth, containerWidthRef] = useContainerWidth(DEFAULT_CHART_WIDTH);
+  const maxLeftLabelsWidth = Math.round((containerWidth ?? 0) / 2);
```

Comment:
`containerWidth` is a number, so I believe `?? 0` is not necessary

Reply:
It is `null | number`

Reply:
`useContainerWidth` returns `[width: number, ref: React.Ref<HTMLDivElement>]`. Anyway this is not critical

Reply:
Ah, sure thing! I didn't even noticed that is not useContainerQuery we have in there which produce a nullable value. Thanks!